### PR TITLE
Added basePath support

### PR DESCRIPTION
**Pull request by kroeder**
Issue:

## What I did

Added support for basePath via webpack plugin that reads the tsconfig.json basePath value of an application

Discussed this with @igor-dv 

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

**Comment by igor-dv** [comment]
CC #2718

File: app/angular/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "@storybook/core": "4.0.0-alpha.22",
     "@storybook/node-logger": "4.0.0-alpha.22",
     "angular2-template-loader": "^0.6.2",
+    "tsconfig-paths-webpack-plugin": "^3.2.0",
```
Comment (igor-dv) [inline app/angular/package.json]: please place it in an alphabetical order

File: app/angular/src/server/angular-cli_config.js [hunk 2]
```diff
@@ -133,6 +134,11 @@ export function applyAngularCliWebpackConfig(baseConfig, cliWebpackConfigOptions
     modules: Array.from(
       new Set([...baseConfig.resolve.modules, ...cliCommonConfig.resolve.modules])
     ),
+    plugins: [
+      new TsconfigPathsPlugin({
+        configFile: cliWebpackConfigOptions.buildOptions.tsConfig,
```
Comment (igor-dv) [inline app/angular/src/server/angular-cli_config.js]: so should we still add the "cliCommonConfig.resolve.modules" ? can you fix then the hardcoded `'src/tsconfig.app.json'` up there ?

Comment (kroeder) [reply app/angular/src/server/angular-cli_config.js]: See my latest commit - Hope I changed the line you meant

**Review by igor-dv — APPROVED** [review]

**Pull request merged**